onedal/spmd/basic_statistics/incremental_basic_statistics.py

```diff
@@ -0,0 +1,72 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from daal4py.sklearn._utils import get_dtype
+
+from ...basic_statistics import (
+    IncrementalBasicStatistics as IncrementalBasicStatistics_nonSPMD,
```

Comment:
There is another option with Base prefix or suffix. But definitely `nonSPMD` should be changed
```suggestion
    IncrementalBasicStatistics as IncrementalBasicStatistics_Base,
```